internal/geometry/triangle_test.go

```diff
@@ -27,12 +27,32 @@ func TestAddXY(t *testing.T) {
 		want3 float64
 		// want []float64
 	}{
-		Triangle{
-			{0, 0},
-			{0, 4},
-			{3, 0},
-		}, {0, 4, 3},
+
+		// 		{
+		// 	ta: Triangle{A: Point{0, 0}, B: Point{0, 4}, C: Point{3, 0}},
+		// 	want1: 0,
+		// 	want2: 0,
+		// 	want3: 0,
+		// },
+		{ta: Triangle{
+			A: Point{X: 0, Y: 0},
+			B: Point{X: 0, Y: 4},
+			C: Point{X: 3, Y: 0},
+		},
+			want1: 0,
+			want2: 4,
+			want3: 3,
+		},
```

Comment:
indent suggestion:
```suggestion
		{
		  ta: Triangle{
			  A: Point{X: 0, Y: 0},
			  B: Point{X: 0, Y: 4},
			  C: Point{X: 3, Y: 0},
		  },
			want1: 0,
			want2: 4,
			want3: 3,
		},
```

Reply:
Why?
It's unclear to me why the triangle values/keys should have more spacing than just a tab. Also why the `want` values/keys should have a lot of spacing, but not sit at on the same position as the values/keys above?
Should I change my spacing/indentation settings?

Reply:
The way I see it is:

```go
var x = {
  a: 1,
  b: 2,
}

// or

var y = { a: 1, b: 2 } 
```

But mixing the two is weird.

e.g.

```
var z = { a: 1, b: 2,
}
```